.github/workflows/release-and-pr.yml

```diff
@@ -0,0 +1,84 @@
+name: Compare ingress version -- Build & publish if new version
```

Comment:
Are we publishing, or creating a PR/candidate release?

Reply:
We are publishing the container on docker hub and the github release/tag. The last piece is creating a PR for the update to the sigsci-values.yaml override file.

Reply:
We could automate the approval and merge of the PR but I though we wanted a little visibility into the process

Reply:
I agree, we do. I'm questioning the descriptiveness of this name for the action: "Compare ingress version -- Build & publish if new version"